app/assets/javascripts/shortcuts_navigation.coffee

```diff
@@ -14,6 +14,7 @@ class @ShortcutsNavigation extends Shortcuts
     Mousetrap.bind('g m', -> ShortcutsNavigation.findAndFollowLink('.shortcuts-merge_requests'))
     Mousetrap.bind('g w', -> ShortcutsNavigation.findAndFollowLink('.shortcuts-wiki'))
     Mousetrap.bind('g s', -> ShortcutsNavigation.findAndFollowLink('.shortcuts-snippets'))
+    Mousetrap.bind('t', -> ShortcutsNavigation.findAndFollowLink('.shortcuts-find-file'))
```

Comment:
Okay, @Razer6 Thank you for your suggestions.
I updated ```_shortcuts.html.haml``` and screenshot.
and I added shortcuts using up and down array key in find file view to select file more conveniently.

The reason that add ```t``` shortcut in ```app/assets/javascripts/shortcuts_navigation.coffee``` is to use find file view other page(eg. merge request/commit/branch page) 
If necessary.

but If not necessary, I just define it on the tree view.

what do you think about that?